test/NameRegistry.t.sol

```diff
@@ -243,7 +252,9 @@ contract NameRegistryTest is Test {
         bytes32 commitHash = nameRegistry.generateCommit("bob", bob, secret, recovery);
         nameRegistry.makeCommit(commitHash);
 
-        vm.warp(block.timestamp + COMMIT_REVEAL_DELAY);
+        delay = delay % REASONABLE_SLA_PERIOD; // test delays up to REASONABLE_SLA_PERIOD && protect against overflow
+        vm.assume(delay > COMMIT_REPLAY_DELAY);
```

Comment:
moving this above like 251 here would allow us to exit the test faster when delay <= COMMIT_REPLAY_DELAY which should marginally speed up the fuzzing tests